src/plone/recipe/zope2instance/recipe.py

```diff
@@ -913,22 +914,51 @@ def build_wsgi_ini(self):
             "profile_flush_at_shutdown": profile_flush_at_shutdown,
             "profile_unwind": profile_unwind,
         }
-        global wsgi_ini_template
+
+        # Check custom wsgi-ini-template and wsgi-logging-ini-template
         wsgi_ini_template_path = self.options.get("wsgi-ini-template")
+        wsgi_logging_ini_template_path = self.options.get("wsgi-logging-ini-template")
+        if wsgi_ini_template_path and wsgi_logging_ini_template_path:
+            raise ValueError(
+                "wsgi_ini_template_path and wsgi_logging_ini_template_path "
+                "cannot be used together."
+            )
+
+        global wsgi_ini_template
+
+        # Load custom wsgi and logging template from file
         if wsgi_ini_template_path:
             try:
                 with open(wsgi_ini_template_path) as fp:
                     wsgi_ini_template = fp.read()
             except OSError:
                 raise
 
+        # Load default global wsgi template and load custom wsgi logging template
+        elif wsgi_logging_ini_template_path:
+            try:
+                with open(wsgi_logging_ini_template_path) as fp:
+                    # Add custom wsgi logging template to wsgi template
+                    wsgi_ini_template += fp.read()
+            except OSError:
+                raise
+
+        # Load default global wsgi and logging template
+        else:
+            global wsgi_logging_ini_template
+            wsgi_ini_template += wsgi_logging_ini_template
```

Comment:
This is a problem if you have multiple buildout parts with this recipe. the template string gets multiplied every time the recipe is called. I'll prep a fix for that.